package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.1.3",
```

Comment:
The package minimist is not on the dependencies so npm install the package and the test are failing on localhost.